### ISSUE TITLE
[ELWCARDS] Card Sharks - Crashes on Initialization

### ISSUE THREAD

**Issue by tuday2** [enusbaum/MBBSEmu#115]
**Module Information**
* Identifier: ELWCARDS
* Module Name: Card Sharks
* Version: 1.1

**Describe the bug**
Crashes on Initialization

**To Reproduce**
1. Add to modules.json
2. Load MBBSEmu

**Expected behavior**
Module loads and is playable

**Screenshots**
```
Critical Exception has occurred:
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'Unknown Exported Function Ordinal in MAJORBBS: 232')
   at MBBSEmu.HostProcess.ExportedModules.Majorbbs.Invoke(UInt16 ordinal, Boolean offsetsOnly)
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.ExternalFunctionDelegate(UInt16 ordinal, UInt16 functionOrdinal)
   at MBBSEmu.CPU.CpuCore.Tick()
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.Module.MbbsModule.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassSetState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.HostProcess.MbbsHost.AddModule(MbbsModule module)
   at MBBSEmu.Program.Run(String[] args)
```

**Software Information:**
- OS: Windows 2019 Server Standard
- Telnet Client: NetRunner

**Comment by enusbaum** [comment]
`MAJORBBS.232` is `FSCANF` which is a libc implementation to acquire formatted input from a specified stream.

[Reference](http://www.cplusplus.com/reference/cstdio/fscanf/)

**Comment by enusbaum** [comment]
@paladine libc routine for you 😄

**Comment by tuday2** [comment]
@enusbaum @paladine This is fixed by PR #183   (module loads), I will work on beating it up and log anything new separately.

Please close.
